Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM alpine:3.4
+FROM mhart/alpine-node
```

Comment:
this image is not an automated build, I cannot determine what is in it or how it was made.
also, you should use a specific tag